src/Resources/config/block.xml

```diff
@@ -46,5 +46,6 @@
             <argument type="service" id="twig"/>
             <argument type="service" id="sonata.page.manager.page"/>
         </service>
+        <service id="Sonata\BlockBundle\Block\BlockServiceManagerInterface" alias="sonata.block.manager"/>
```

Comment:
I think this alias should be something like "sonata.page.manager.block" to not collide with something declared on blockbundle.

Reply:
I did an alias to: "sonata.page.block.service_manager". Other option is to create the alias inside blockbundle, but that's not how we did it in other cases like mediabundle aliasing things of classification bundle, wdyt @VincentLanglet ?

(we can change it later anyway)

Reply:
I can't change this alias, should we define it inside blockbundle @VincentLanglet ?